Alicerce/Sources/Logging/Log+StringLogDestination.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Log+StringLogDestination.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class StringLogDestination: LogDestination {
+
+        private static let dispatchQueueLabel = "com.mindera.Alicerce.StringLogDestination"
+
+        public private(set) var dispatchQueue: DispatchQueue
+        public private(set) var minLevel: Level
+        public private(set) var formatter: LogItemFormatter
+        public var output = ""
+        public var linefeed = "\n"
+
+        //MARK:- lifecycle
+
+        public init(minLevel: Level = Log.Level.error,
+                    formatter: LogItemFormatter = Log.StringLogItemFormatter(),
+                    dispatchQueue: DispatchQueue = DispatchQueue(label: StringLogDestination.dispatchQueueLabel)) {
+
+            self.minLevel = minLevel
+            self.formatter = formatter
+            self.dispatchQueue = dispatchQueue
+        }
+
+        //MARK:- public methods
+
+        public func write(item: Item, completion: @escaping (LogDestination, Log.Item, Error?) -> Void) {
+            dispatchQueue.async { [weak self] in
+                guard let strongSelf = self else { return }
+
+                var reportedError: Error?
```

Comment:
I might be missing something again, but I don't see `reportedError` being changed anywhere?